README.md

```diff
@@ -9,26 +9,36 @@ for better realtime performance
 the easy creation of new controllers
 5. The `MjbotsRobotInterface` which provides a convenient interface for communicating with any number
 of moteus motor controllers 
+6. The `RobotInterface` which provides and interface for updating robot state and joint torques.
 
 Note: This library only supports torque commands. If you wish to use
 position control, you either must close the loop yourself or modify the 
 library to allow for the position loop to Run on the moteus.
 
-#Important
+# Important
 In order to keep the message size down, kp and kd on the motors must be set to 0
 
 # Usage
+
 ## MjbotsControlLoop:
-To use the `MjbotsControlLoop` create a class which inherits the `MjbotsControlLoop`
-and implements `CalcTorques` to set the torques in the robot object. 
+To use the Mjbots control loop, create a class which inherits the 
+`MjbotsControlLoop` object and implements `CalcTorques` to set the torques in 
+the robot object as follows.
+
+```cpp
+class SimpleRobotControlLoop : public kodlab::mjbots::MjbotsControlLoop
```

Comment:
SimpleRobot is a derived class and should either be in the template or this default version should just be RobotControlLoop.